packages/components/src/icon-button/style.scss

```diff
@@ -1,15 +1,4 @@
 .components-icon-button {
-	display: flex;
-	align-items: center;
-	padding: 8px;
-	margin: 0;
-	border: none;
-	background: none;
-	color: $dark-gray-500;
```

Comment:
Removing this is possibly what caused the regression. If the color shouldn't live here, then it needs to live elsewhere so the button can inherit it, for example using `currentColor`.

Reply:
My main issue is that the color shouldn't be different between Button  and IconButton so we should move this to the Button stylesheet.

Reply:
Fix here https://github.com/WordPress/gutenberg/pull/19123